src/contexts/states/statistics.ts

```diff
@@ -0,0 +1,8 @@
+export const initStatistics: State.Statistics = {
+  tip_block_number: '0',
```

Comment:
is this state tip_blok_number the same with states/index.ts tipBlockNumber above?

Reply:
No. This `tip_block_number` is in statistic data type which come from statistic rpc,  and the other `tipBlockNumber` is from another rpc which only contains `tipBlockNumber`.

Reply:
No. This tip_block_number is from statistic rpc which contains many statistic data, and ` states/index.ts tipBlockNumber` is from another rpc which only contains tipBlockNumber.

Reply:
https://github.com/nervosnetwork/ckb-explorer-frontend/pull/164/commits/412597c06e3d0cb46a2a417e6d80f66792c8d1fe